hezar/models/__init__.py

```diff
@@ -1,10 +1,10 @@
-from .model import Model
```

Comment:
@HamedBabaei jan, base classes must be on top of the `__init__` files. You can ignore sorting here.

Reply:
Hey @arxyzan jan, is there any reason for this? if this is a personal preference of having base classes on top, we might need to use the `# isort: skip` command for that specific import. If this is not we might consider ignoring sorting in `__init__` files!

Reply:
The main reason is that we faced some issues with cyclic imports in the past due to this. Having the base class imported on top of the `__init__` file solves the issue.

Reply:
I see, then I just put `# isort: skip` for base classes, it will make it easier to follow up. thanks!